### PR TITLE
prefill bookmarklet with markdown-formatted content

### PR DESCRIPTION
**Pull request by Raven24**
this is based on #5384 and also adds some calculations to center the bookmarklet in the parent browser window.
the inline JS is now a bit longer, but hopefully more readable.

### CODE REVIEW

File: app/assets/javascripts/app/views/bookmarklet_view.js [hunk 1]
```diff
@@ -25,8 +25,12 @@ app.views.Bookmarklet = Backbone.View.extend({
     var p = this.param_contents;
     if( p.content ) return p.content;
 
-    var contents = p.title + this.separator + p.url;
-    if( p.notes ) contents += this.separator + p.notes;
+    var contents = "### " + p.title + this.separator;
```
Comment (Raven24) [inline app/assets/javascripts/app/views/bookmarklet_view.js]: I'd be okay with simply using bold text instead of a heading... opinions?

**Comment by jhass** [comment]
Mh, I wonder if we can externalize this into a file and call uglifier on it.

**Comment by jhass** [comment]
Or yet better hook it into the asset pipeline and read the compiled file, like I hooked generating the static HTML pages into it, would avoid the execjs dependency on runtime that uglifier would add, mh.

**Comment by Raven24** [comment]
ok, gotta have a look at how you did that :)

**Comment by jhass** [comment]
yeah, it might be overkill, but that inline JS...

**Comment by Raven24** [comment]
alright, I played around with asset compilation a bit, and I think running uglifier manually is the easiest way in this case. I added that step to the precompile rake task, while also keeping the JS updated in development.

**Comment by jhass** [comment]
Nice!

**Comment by svbergerem** [comment]
@Raven24 Nice! :) I just found #3076 where you wrote that you didn't want to touch the bookmarklet code. Since you touched the code in this PR it would be great if you could limit the string length and also fix that issue. (Obviously it's up to you if you would like to do that)

**Comment by Raven24** [comment]
right, totally forgot about that one (well, it's 3 years old...)
sure, I can limit the GET length to something reasonable, while I'm at it

**Comment by jhass** [comment]
Thank you!

**Pull request merged**